src/components/x-modal.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <BaseEventsModal :eventsToOpenModal="openEvents" :animation="animation">
     <Mobile v-if="$x.device === 'mobile'" />
-    <Desktop v-else />
+    <Desktop v-else class="x-desktop" />
```

Comment:
add also the class to mobile

Reply:
I didn't add it because is already added in another PR and in this one is not needed.